externs/shaka/player.js

```diff
@@ -1095,3 +1095,21 @@ shaka.extern.LanguageRole;
  * @exportDoc
  */
 shaka.extern.Thumbnail;
+
+
+/**
+ * @typedef {{
+ *   payload: string,
+ *   startTime: number,
+ *   endTime: number
+ * }}
+ *
+ * @property {string} payload
+ *    The text payload of the chapter.
```

Comment:
The WebVTT spec calls this the "title" of the chapter.  Can we please use the same wording here instead of "payload", which is harder to interpret?

Reply:
Done!